### ISSUE TITLE
Feature Request: Add easier way for accessing createGoalExport() from other applications

### ISSUE THREAD

**Issue by JustBru00** [Joshimuz/mcbingo#30]
Perhaps for example: `https://minecraftbingo.com/?s=3-0-0-3_76812&export=json`

This allows a standard way to add other export types.
The page that would load would just be plain text with the json generated by the createGoalExport() method.

This will allow easy access via other software with a simple https GET request.

Thanks for making this awesome website,
Justin B.

**Comment by Joshimuz** [comment]
A good idea!
Next time I work on the site (probably around 1.17 launch) I'll try to do something like this

**Comment by tduva** [comment]
Good idea, unfortunately the bingo sheet is generated entirely clientside in JS. While it would be possible to have a URL parameter that shows only the JSON export, a HTTP GET request would only give you the website's source, which would then have to be run in a browser to generate the sheet and get the actual JSON.

So not feasible without a rewrite and different hosting that supports serverside code.
